casual/casual-caller/src/main/java/se/laz/casual/connection/caller/FailoverAlgorithm.java

```diff
@@ -86,11 +87,27 @@ private List<ConnectionFactoryEntry> getFoundAndValidEntries(ConnectionFactoryLo
     private <T> T issueCall(String serviceName, List<ConnectionFactoryEntry> validEntries, FunctionThrowsResourceException<CasualConnection, T> doCall)
     {
         Exception thrownException = null;
+
+        // Sticky transaction handling
+        try
+        {
+            Optional<T> stickyMaybe = handleTransactionSticky(serviceName, validEntries, doCall);
+
+            if (stickyMaybe.isPresent()) return stickyMaybe.get();
```

Comment:
Prefer to never use single line ifs, always enclose in `{` `}` to prevent mistakes later.

Reply:
I'll change it. There are a few places where I've been a bit sloppy with this